app/features-json/setup_json_controller.rb

```diff
@@ -8,5 +8,19 @@ class SetupJSONController < APIController
     get "#{HOME}/configured", authenticate: false do
       return json(Services.onboarding_service.correct_setup?)
     end
+
+    get "#{HOME}/userDetails" do
+      code_hosting_service = current_user_config_service.code_hosting_service(
+        provider_credential: Services.provider_credential
```

Comment:
`Services.provider_credential` is the bot provider credential. I think you want the current user instead, right?

Reply:
Ah yeah, seems like I had the same user for both the bot and the user. Do you know how I can access the provider credentials of the user in a JSON controller?